seneca.js

```diff
@@ -796,6 +797,22 @@ function make_seneca (initial_options) {
     return pins
   }
 
+  // Perform an action, if the action exists. Don't kill seneca, unless the user
+  // provides a fatal$ flag.
+  function api_publish () {
+    var self = this
+
+    var spec = Common.parsePattern(self, arrayify(arguments), 'done:f?')
+    var args = spec.pattern
+    var actdone = spec.done
+
+    if (!args.fatal$) {
+      args.fatal$ = false
+    }
+
+    return self.act(args, actdone)
```

Comment:
2 thoughts:

It might be nice to give the user the chance to use a result if it exists.

and we don't gain much by swallowing it, do we? the user can ignore it, if they want to.